extensions/amp-story-shopping/0.1/amp-story-shopping.md

```diff
@@ -58,7 +58,7 @@ They are positioned [`absolute`](https://developer.mozilla.org/en-US/docs/Web/CS
 ```
 
 Example of positioning an [`amp-story-shopping-tag`](#amp-story-shopping-tag) and testing responsiveness:
-<amp-img alt="An example of positioning an `amp-story-shopping-tag` and teseting responsiveness" src="https://user-images.githubusercontent.com/3860311/155751130-558b5ab6-1db3-4ca7-b913-4be4761fdb29.gif" layout="intrinsic" width="840" height="543">
+<amp-img alt="An example of positioning an `amp-story-shopping-tag` and teseting responsiveness" src="https://user-images.githubusercontent.com/3860311/155751130-558b5ab6-1db3-4ca7-b913-4be4761fdb29.gif" layout="intrinsic" width="840" height="543"></amp-img>
```

Comment:
Do we need the single quote? `` ` ``amp-story-shopping-tag`` ` `` as opposed to `amp-story-shopping-tag`?

Also, theres a typo:
teseting -> testing